tests/avro/mock_registry.py

```diff
@@ -182,8 +182,3 @@ def shutdown(self):
         if self.server:
             self.server.shutdown()
             self.server.socket.close()
-
-
-if __name__ == '__main__':
```

Comment:
I'm not entirely sure why this would have ever be used. Can you think of why would someone want to start a mock registry server on port `0` by running this file? Shouldn't this be done inside of a specific test or class of tests?